src/toolkits/drawing_classifier/dc_data_iterator.hpp

```diff
@@ -51,10 +52,8 @@ class data_iterator {
      *
      * If empty, then the labels will be inferred from the data. If non-empty,
      * an exception will be thrown upon encountering an unexpected label.
-     *
-     * \TODO: This should be a flex_list to accomodate integer labels!
```

Comment:
okay, you already said, it's a feature change. The underlying representation of class labels is changed.

Reply:
Not being able to support integer targets was surfaced by adding the stroke-based model to our Python unit tests (the stroke-based data we generated used integer targets). Since those changes were not that code-heavy, I made those changes as part of this PR. Since we supported integer targets in 5.8, we must also support them in 6.0.